packages/spotlight/Spottable/Spottable.js

```diff
@@ -222,6 +223,18 @@ const Spottable = hoc(defaultConfig, (config, Wrapped) => {
 			this.node = ReactDOM.findDOMNode(this);
 		}
 
+		shouldComponentUpdate (nextProps, nextState) {
+			const mutatedState = {...this.state};
```

Comment:
We're incurring some additional cost by cloning both state objects just to delete the `focused` member. Would it be better to compare the state members individually since there is a small known set? Also, is it necessary to keep `selectionKeys` in state? I couldn't see a reason to copy it into state. Removing it would simplify the state comparison further.

Reply:
I think that's a good observation - I don't see where it's necessary to keep `selectionKeys` in state, since we won't need to compare them against previous props in `gDSFP`. I'm removing this and updating the `sCU` logic. 👍